src/Apps/getAppRoutes.tsx

```diff
@@ -27,5 +28,8 @@ export function getAppRoutes(): RouteConfig[] {
     {
       routes: searchRoutes,
     },
+    {
+      routes: conversationRoutes,
+    },
```

Comment:
#superminor this object is alpha

Reply:
what do you mean?

Reply:
Oh sorry :) meaning everything is intentionally alphabetized so that things are easier to find later.